ldoc.lua

```diff
@@ -30,7 +30,7 @@ utils.unpack = utils.unpack or unpack or table.unpack
 local append = table.insert
 local lapp = require 'pl.lapp'
 
-local version = '1.4.6'
+local version = '1.4.7'
```

Comment:
Please don't bump the version number as part of PR's, well do that right before an actual release.